lefthook.yml

```
@@ -9,8 +9,8 @@ pre-commit:
       run: golangci-lint run --config .golangci.yml
-#
-#pre-push:
-#  commands:
-#    docs:
-#      run: swag init --generalInfo ./cmd/main.go --output ./docs --parseDependency --parseInternal && git add ./docs
\ No newline at end of file
+
+pre-push:
+  commands:
+    docs:
+      run: swag init --generalInfo ./cmd/main.go --output ./docs --parseDependency --parseInternal && git add ./docs
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)